### PR TITLE
Update RWCloud user docs (GA)

### PR DESCRIPTION
**Pull request by CharlieSYH**
## Info

This is the user documentation update for cloud GA.

## For reviewers

- **Preview**

  https://pr-998.d2fbku9n2b6wde.amplifyapp.com/cloud/intro/

- **Key points**

  [ Parts that may need revision or extra consideration. ]

## Before merging

- [ ] I have checked the doc site preview, and the updated parts look good.

- [ ] I have acquired the approval from the owner (and optionally the reviewers) of the code PR and at least one tech writer (`bernscode`, `CharlieSYH`, `emile-00`, & `hengm3467`).

### CODE REVIEW

File: cloud/source-create-a-source.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+6. Check the generated SQL statement and click **Confirm** to create the source in your database.
+
+    <img
+    src={require('./images/source-create-kafka-4.png').default}
+    alt="Create source - Kafka - step 4"
+    />
+
+## Using SQL command
+
+Refer to [`CREARE SOURCE`](https://www.risingwave.dev/docs/current/sql-create-source/#supported-sources) in the RisngWave Database documentation. Select a connector to see the SQL syntax, options, and sample statement of connecting RisingWave Database to the connector.
```
Comment (hengm3467) [inline cloud/source-create-a-source.md:47]: The last sentence is from the `CREATE SOURCE` page. I think we'd want to revise it to be more accurate, something like: Select a source to see the ... statement of connecting RisingWave Database to the source. 

And the row header should also be changed from "Connector" to "Source". What do you think? @CharlieSYH

**Review by hengm3467 — APPROVED** [review]
Just added one comment. Otherwise looks good to me. Good work! @CharlieSYH

**Review by Ad-Bean — APPROVED** [review]
LGTM, though there might be some improvements:
1. on the page https://pr-998.d2fbku9n2b6wde.amplifyapp.com/cloud/manage-clusters/, https://pr-998.d2fbku9n2b6wde.amplifyapp.com/cloud/manage-sources/ and https://pr-998.d2fbku9n2b6wde.amplifyapp.com/cloud/manage-database-users/, cannot go to the next page
2. https://pr-998.d2fbku9n2b6wde.amplifyapp.com/cloud/manage-your-account/ directs to the wrong next page, and the sidebar cannot highlight the right tab
3. should we add a zoom-in plug for the pictures?

File: cloud/source-create-a-source.md (hunk 1 above)
Comment (CharlieSYH) [reply cloud/source-create-a-source.md:47]: Agree. I'll update the database docs in this PR.

**Comment by CharlieSYH** [comment]
@Ad-Bean Nice catch!
> 1. on the page https://pr-998.d2fbku9n2b6wde.amplifyapp.com/cloud/manage-clusters/, https://pr-998.d2fbku9n2b6wde.amplifyapp.com/cloud/manage-sources/ and https://pr-998.d2fbku9n2b6wde.amplifyapp.com/cloud/manage-database-users/, cannot go to the next page

This issue might be caused by adding a doc link to a category (so that when clicking the category, users can see the overview page and expand the sidebar at the same time). 
image

> 2. https://pr-998.d2fbku9n2b6wde.amplifyapp.com/cloud/manage-your-account/ directs to the wrong next page, and the sidebar cannot highlight the right tab

Actually, the next page is correct here because tabs are not actual pages. And please try out the sidebar again. I made some minor adjustments.

> 3. should we add a zoom-in plug for the pictures?

I want that. But it'd better be a properly designed plugin.

Thanks a lot.

**Comment by CharlieSYH** [comment]
Merging this PR because some links on the Portal rely on it. Reviewers can still comment on this PR to propose edits.

**Pull request merged**